qcodes/instrument_drivers/stanford_research/SR830.py

```diff
@@ -472,6 +472,76 @@ def parse_offset_get(s):
         self._buffer2_ready = False
 
         self.connect_message()
+    
+    
+    SNAP_PARAMETERS = {
+            'x': '1',   
+            'y': '2',  
+            'r': '3', 
+            'p': '4',   
+        'phase': '4',  
+           'θ' : '4',
+         'aux1': '5',  
+         'aux2': '6',   
+         'aux3': '7',  
+         'aux4': '8',  
+         'freq': '9',   
+          'ch1': '10',
+          'ch2': '11'  
+    }
+    
+    def snap(self, *parameters: str) -> Tuple[float, ...]:
+        """
+Get between 2 and 6 parameters at a single instant. This provides a coherent
```

Comment:
so far as i know the text for the docstring shall start from `"""`, like this:
```
    def smth():
        """
        a great docstring
        ...
```

Reply:
In addition our docstrings should be valid restructured text using the Google docs extension see https://qcodes.github.io/Qcodes/examples/Creating%20Instrument%20Drivers.html#Documentation and https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html

Specifically the args, returns and example sections should conform to the standard and you are not allowed to create other sections like the Units and Limitations. You probably want to put this in a notes section